src/topology/algebra/group.lean

```diff
@@ -1032,6 +1032,48 @@ begin
   exact topological_group.t1_space (G ⧸ S) ((quotient_map_quotient_mk.is_closed_preimage).mp hS),
 end
 
+/-- A discrete subgroup of a topological group `G` acts on `G` properly discontinuously on the left.
+-/
```

Comment:
I think it is worth avoiding the word "discrete" here since there are two possible meanings. How about:
```suggestion
/-- Given a subgroup `S` of topological group `G`, if `S ∩ K` is finite for all compact `K`, then `S` acts on `G` properly discontinuously on the left.
-/
```
The second meaning is that one might mean `S` gets the discrete topology. Of course they're related: I _think_ the following is true:
```lean
example {α : Type*} [topological_space α] [locally_compact_space α] [t2_space α]
  (s : set α) (hs : is_closed s) :
  tendsto (coe : s → α) filter.cofinite (cocompact α) ↔ discrete_topology s :=
```
but missing from Mathlib.

See also https://github.com/leanprover-community/mathlib/blob/b381419c18566398f9a7f92c75a18256519ce928/src/topology/instances/real.lean#L233 vs https://github.com/leanprover-community/mathlib/blob/b381419c18566398f9a7f92c75a18256519ce928/src/analysis/normed_space/basic.lean#L165

Reply:
I actually would like to leave in the word "discrete", because it's clearly related, and someone might search mathlib using that term.  How about a phrasing which makes clear that this is not necessarily the standard use, like
> A subgroup `S` of topological group `G` acts on `G` properly discontinuously on the left, if it is discrete in the sense that `S ∩ K` is finite for all compact `K`.  (See also `discrete_topology`.)

And indeed, someone should figure out and Lean-ify what the relationship between the concepts is.

Reply:
Perfect; go for it.